lib/wallet/src/Cardano/Wallet/Primitive/Types/Tx/SealedTx.hs

```diff
@@ -179,6 +185,18 @@ cardanoTxIdeallyNoLaterThan
     -> InAnyCardanoEra Cardano.Tx
 cardanoTxIdeallyNoLaterThan era = unsafeCardanoTx . ideallyNoLaterThan era
 
+-- | Re-deserialises the bytes of the 'SealedTx' as a transaction in the
+-- provided era, and that era only.
+cardanoTxInExactEra
```

Comment:
I'd name this function `unsealTransaction` or `unsealTx` . The fact that its in the exact era is already captured by the type, so no need to express it in name.

Reply:
I'd also vote for `unsealTx`, provided that this really is a form of "unsealing".

(The `Tx` shorthand is already prevalent throughout our code, so we might as well take advantage of this common shorthand.)

Reply:
I feel this function is too weird to simply be called `unsealTx` plus there's symmetry with `cardanoTxIdeallyNoLaterThan`, so keeping.

Perhaps `unsealTx` will be a good name if we rethink `SealedTx` in the future and/or if we remove `cardanoTxIdeallyNoLaterThan`.